Makefile

```diff
@@ -9,13 +12,13 @@ vendor: composer.json composer.lock
 build/cache:
 	mkdir -p build/cache
 
-php-cs-fixer-v2.phar:
-	wget -nc http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar
-	chmod a+x php-cs-fixer-v2.phar
+$(PHP-CS-FIXER):
+	wget http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar
+	chmod a+x $(PHP-CS-FIXER)
 
-phpstan.phar:
-	wget -nc https://github.com/phpstan/phpstan/releases/download/0.9.1/phpstan.phar
-	chmod a+x phpstan.phar
+$(PHPSTAN):
+	wget https://github.com/phpstan/phpstan/releases/download/0.9.1/phpstan.phar
+	chmod a+x $(PHPSTAN)
```

Comment:
Same here. That's the targets you define to download the default tools. If someone wants to use other version of a tool, it is their responsibility to download it.